### PR TITLE
Upgrade to Symfony 6

### PR DESCRIPTION
**Pull request by chapterjason**
- enhance: Update readme
- enhance: Upgrade to symfony 6
- enhance: Update symfony/framework-bundle recipe
- enhance: Update symfony/routing recipe
- enhance: Add context property type

- [ ] Update dependency; wait for https://github.com/yceruto/ddd-maker-bundle/pull/4

### CODE REVIEW

File: config/routes.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+kernel:
+    resource: ../context/Kernel.php
+    type: annotation
```
Comment (yceruto) [inline config/routes.yaml]: I'd remove this routes config, I don't expect anyone using the kernel class as controller on this skeleton

**Review by yceruto — CHANGES_REQUESTED** [review]
Thanks for this upgrade! I will take care of reviewing the other PRs.

File: composer.json [hunk 2]
```diff
@@ -4,15 +4,15 @@
...
         "ext-ctype": "*",
         "ext-iconv": "*",
-        "symfony/console": "5.*",
-        "symfony/dotenv": "5.*",
+        "symfony/console": "6.*",
+        "symfony/dotenv": "6.*",
         "symfony/flex": "^1.17",
-        "symfony/framework-bundle": "5.*",
+        "symfony/framework-bundle": "6.*",
         "symfony/runtime": "*",
```
Comment (chapterjason) [inline composer.json]: @yceruto Is this correct?
Shouldn't it be:

```suggestion
        "symfony/runtime": "6.*",
```

**Comment by yceruto** [comment]
Thank you @chapterjason!

**Pull request merged**